src/bentoml/_internal/frameworks/diffusers.py

```diff
@@ -405,19 +402,21 @@ def get_runnable(bento_model: bentoml.Model) -> t.Type[bentoml.Runnable]:
     Private API: use :obj:`~bentoml.Model.to_runnable` instead.
     """
 
-    partial_kwargs: t.Dict[str, t.Any] = bento_model.info.options.partial_kwargs  # type: ignore
+    bento_options: DiffusersOptions = t.cast(DiffusersOptions, bento_model.info.options)
+    partial_kwargs: t.Dict[str, t.Any] = bento_options.partial_kwargs  # type: ignore
     pipeline_class: type[diffusers.DiffusionPipeline] = (
-        bento_model.info.options.pipeline_class or diffusers.StableDiffusionPipeline
+        bento_options.pipeline_class or diffusers.StableDiffusionPipeline
     )
     scheduler_class: type[
         diffusers.SchedulerMixin
-    ] | None = bento_model.info.options.scheduler_class
-    custom_pipeline: str | None = bento_model.info.options.custom_pipeline
-    _enable_xformers: str | None = bento_model.info.options.enable_xformers
-    enable_attention_slicing: int | str | None = (
-        bento_model.info.options.enable_attention_slicing
-    )
-    _torch_dtype: str | torch.dtype | None = bento_model.info.options.torch_dtype
+    ] | None = bento_options.scheduler_class
+    custom_pipeline: str | None = bento_options.custom_pipeline
+    _enable_xformers: bool | None = bento_options.enable_xformers
+    enable_attention_slicing: int | str | None = bento_options.enable_attention_slicing
+    _torch_dtype: str | torch.dtype | None = bento_options.torch_dtype
+    device_map: str | dict[
+        str, int | str | torch.device
+    ] | None = bento_options.device_map
```

Comment:
I don't think you need to hint variable here, since you already cast the `bento_options` to DiffuserOptions

Reply:
done